client/src/components/Userpage.js

```diff
@@ -0,0 +1,243 @@
+    },
+    {
+      location: "Chichibu",
+      country: "Japan",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: true,
+    },
+    {
+      location: "Copenhagen",
+      country: "Denmark",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: true,
+    },
+    {
+      location: "Vancouver",
+      country: "Canada",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: true,
+    },
+    {
+      location: "Stockholm",
+      country: "Sweden",
+      img:
+        "https://www.worldatlas.com/r/w960-q80/upload/a1/28/76/shutterstock-133005938.jpg",
+      liked: false,
+    },
+  ]);
+
+  const listOfExploreCards = exploreCards.map((exploreCard) => {
+    if (exploreCard["liked"] === true) {
+      if (invisibleCount < 2) {
+        invisibleCount++;
+        return (
+          <Grid item xs={3} className={classes.exploreContainer}>
+            <ExploreCard
+              location={exploreCard.location}
+              country={exploreCard.country}
+              img={exploreCard.img}
+              onLike={() => {
+                exploreCard["liked"] = false;
+              }}
+            ></ExploreCard>
+          </Grid>
+        );
+      }
+
+      if (invisibleCount > 1) {
+        invisibleCount = 1;
+        return (
+          <React.Fragment>
+            <Grid item xs={3} className={classes.invisible}>
+              <ExploreCard location="Cancun" country="Mexico"></ExploreCard>
+            </Grid>
+            <Grid item xs={3} className={classes.invisible}>
+              <ExploreCard location="Cancun" country="Mexico"></ExploreCard>
+            </Grid>
+            <Grid item xs={3} className={classes.exploreContainer}>
+              <ExploreCard
+                location={exploreCard.location}
+                country={exploreCard.country}
+                img={exploreCard.img}
+                onLike={() => {
+                  exploreCard["liked"] = false;
+                }}
+              ></ExploreCard>
+            </Grid>
+          </React.Fragment>
+        );
+      }
+    }
+  });
+
+  return (
+    <Grid container spacing={0} className={classes.grid}>
+      <Grid container spacing={1} className={classes.leftQuadrant}>
+        <Grid item xs={12} className={classes.container}>
+          <paper className={classes.alignment}>
+            <Avatar />
+          </paper>
+        </Grid>
+        <Grid item xs={12} className={classes.alignment}>
+          <Typography variant="h5" gutterBottom>
+            John Smith
+          </Typography>
+          <Typography className={classes.email} gutterBottom>
+            johnsmith1@gmail.com
+          </Typography>
+        </Grid>
+        <Grid item xs={12} />
+
+        <Button
+          variant="outlined"
+          style={{ color: "#BEC1E2" }}
```

Comment:
We want to be consistent. If doing a style, use `className`.
Also this is the purple colour, so you can simply do 
```
<Button
     variant="outlined"
     color="secondary"
     className={classes.alignment}
>
     Edit
</Button>
```